phpcr-shell/installation.rst

```diff
@@ -35,24 +35,17 @@ If you are using a Symfony2 application and a version of `DoctrinePHPCRBundle
 <https://github.com/doctrine/DoctrinePHPCRBundle/>`_ greater than 1.2 then you
 can easily integrate the PHPCR-Shell.
 
-Simply add the shell to your ``composer.json`` file
+Simply add the shell with composer:
 
-.. code-block:: javascript
+.. code-block:: bash
 
-    { 
-        ...
-        require: {
-            ...
-            "phpcr-shell": "<latest version here>"
-        }
-        ...
-    }
+    $ composer require phpcr/phpcr-shell
```

Comment:
you are right. http://symfony.com/doc/current/book/installation.html#installing-composer-globally recommends this way